metamath-knife/src/main.rs

```diff
@@ -155,7 +156,14 @@ fn main() {
         if matches.is_present("axiom_use") {
             File::create(matches.value_of("axiom_use").unwrap())
                 .and_then(|file| {
-                    db.write_stmt_use(|label| label.starts_with(b"ax-"), &mut BufWriter::new(file))
+                    db.write_stmt_use(
+                        |sref| {
+                            sref.statement_type() == StatementType::Axiom
+                                && sref.proof_slice_at(0) == b"|-"
```

Comment:
Why not use the `$j syntax '|-' as 'wff';` hint?

Reply:
Possibly the instant motivation was that set.mm/miu.mm also uses the popular token |- but doesn't have any $j comments or distinguish the introduction of new non-syntax/non-definition rules with labels that begin with ax-, so this meant to be an incremental improvement with respect to the set.mm databases/examples in the Metamath book.